git-hook

```diff
@@ -27,16 +27,16 @@ which errcheck >/dev/null || (echo "Unable to find errcheck, please install it:
 which golint >/dev/null || (echo "Unable to find golint, please install it: \`go get -u github.com/golang/lint/golint\`" && exit 1)
 
 echo "*** Running \033[0;34mErrcheck\033[0m ***" && \
-for dir in *; do
+for dir in $MODIFIED_DIRS; do
   errcheck github.com/getlantern/$dir || \
-    echo "\033[0;31merrcheck returned error analyzing the package '$dir'\033[0m" && \
+    (echo "\033[0;31merrcheck returned error analyzing the package '$dir'\033[0m" && \
```

Comment:
Thank you! A very tiny detail, though. Could you please place the `(` as in the other cases? (or change the other cases if you prefer). Even though is a tiny style detail, Bash is already a quirky language, so uniformity pays off in my opinion.